### ISSUE TITLE
vue/no-multi-spaces does not report on `<script    >`

### ISSUE THREAD

**Issue by golopot** [vuejs/eslint-plugin-vue#877]
**Tell us about your environment**	
* **ESLint version: 5.6.0**
* **eslint-plugin-vue version: 5.2.2**
* **Node version: 10.5.0**

**Please show your full configuration:**

```js
{
    extends: ['plugin:vue/recommended'],
}
```

**What did you do?**	

```vue
<template>
  <div></div>
</template>

<script   ></script>

<style    ></style>
```

**What did you expect to happen?**
vue/no-multi-spaces reports errors.

**What actually happened?**
No errors.

This bug also affects `vue/html-closing-bracket-spacing` .

**Comment by FloEdelmann** [comment]
Why did you close this issue? I think it's still a valid request.

* `<template >`, `<template lang="html" >` and `</template >` are reported

---

* `<template lang="pug" >` is not reported
* `<script >` and `</script >` are not reported
* `<style >` and `</style >` are not reported
* top-level `<i18n >` and `</i18n >` are not reported
